ironmon_tracker/screens/CustomExtensionsScreen.lua

```diff
@@ -0,0 +1,262 @@
+CustomExtensionsScreen = {
+	Labels = {
+		header = "Custom Extensions",
+		pageFormat = "Pg. %s/%s", -- e.g. Pg. 1/3
+		noExtensions = "No custom extensions found",
+	},
+	Colors = {
+		text = "Default text",
+		border = "Upper box border",
+		boxFill = "Upper box background",
+	},
+}
+
+CustomExtensionsScreen.Pager = {
+	Buttons = {},
+	currentPage = 0,
+	totalPages = 0,
+	defaultSort = function(a, b) return a.extension.name < b.extension.name end, -- Order ascending by extension name
+	realignButtonsToGrid = function(self, x, y, colSpacer, rowSpacer)
+		table.sort(self.Buttons, self.defaultSort)
+		local cutoffX = Constants.SCREEN.WIDTH + Constants.SCREEN.RIGHT_GAP - Constants.SCREEN.MARGIN
+		local cutoffY = Constants.SCREEN.HEIGHT - 20
+		local totalPages = Utils.gridAlign(self.Buttons, x, y, colSpacer, rowSpacer, true, cutoffX, cutoffY)
+		self.currentPage = 1
+		self.totalPages = totalPages or 1
+		CustomExtensionsScreen.Buttons.CurrentPage:updateText()
+	end,
+	getPageText = function(self)
+		if self.totalPages <= 1 then return "Page" end
+		local buffer = Utils.inlineIf(self.currentPage > 9, "", " ") .. Utils.inlineIf(self.totalPages > 9, "", " ")
+		return buffer .. string.format(CustomExtensionsScreen.Labels.pageFormat, self.currentPage, self.totalPages)
+	end,
+	prevPage = function(self)
+		if self.totalPages <= 1 then return end
+		self.currentPage = ((self.currentPage - 2 + self.totalPages) % self.totalPages) + 1
+	end,
+	nextPage = function(self)
+		if self.totalPages <= 1 then return end
+		self.currentPage = (self.currentPage % self.totalPages) + 1
+	end,
+}
+
+CustomExtensionsScreen.Buttons = {
+	EnableCustomExtensions = {
+		type = Constants.ButtonTypes.CHECKBOX,
+		text = " Allow custom code to run", -- offset with a space for appearance
+		clickableArea = { Constants.SCREEN.WIDTH + Constants.SCREEN.MARGIN + 4, Constants.SCREEN.MARGIN + 14, Constants.SCREEN.RIGHT_GAP - 12, 8 },
+		box = {	Constants.SCREEN.WIDTH + Constants.SCREEN.MARGIN + 4, Constants.SCREEN.MARGIN + 14, 8, 8 },
+		toggleState = true, -- update later in initialize
+		toggleColor = "Positive text",
+		onClick = function(self)
+			-- Toggle the setting and store the change to be saved later in Settings.ini
+			self.toggleState = not self.toggleState
+			Options.updateSetting("Enable custom extensions", self.toggleState)
+			Options.forceSave()
+		end
```

Comment:
I had briefly considered something simple like this where it would just grey things out. I like you proposed solution and will give it a go and implement it.

I had also considered this sort of grey-out for cases where you had an extension but it got removed somewhere, then it would still appear in this list and inform you that it couldnt be found (or used). But I didn't implement this yet as there isn't a good way to get extra details on an extension yet, and instead opted to just clear out any extensions that no longer exist in your `Custom` folder, perhaps because you removed the script files on purpose.